README.md

```diff
@@ -3,3 +3,11 @@
 Connect.gov was a precursor to login.gov. Occasionally, people looking for ConneCT, a service offered by the state government of Connecticut will end up on connect.gov.
 
 This repo contains some files to host at connect.gov to disambiguate the 2 services.
+
+## Local Development
+
+To run the server locally:
```

Comment:
Should we add a note here about having `SimpleHTTPServer` installed?

Reply:
It's a python built-in, so I think it's fine?